corona-ide-main/src/main/java/com/coronaide/main/Main.java

```diff
@@ -49,7 +58,7 @@ public static void main(String[] args) {
      * @param args
      *            Arguments used to control application behavior
      */
-    public void start(String[] args) {
+    public void run(String[] args) {
```

Comment:
had to rename this because the JavaFX application overridden class is named start. I didn't originally had the JavaFX application be a separate class, but it was causing a lot more pain than necessary so I bit the bullet and moved it here.